tools/Makefile

```diff
@@ -129,7 +112,7 @@ ifeq ($(HOST_OS),Darwin)
 else
   $(curdir)/dwarves/compile += $(curdir)/elfutils/compile
   $(curdir)/elfutils/compile := $(curdir)/m4/compile $(curdir)/zlib/compile
-  tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS)$(CONFIG_DWARVES),y) += dwarves
+  tools-$(if $(CONFIG_TOOLS_BUILD_ALL)$(CONFIG_DWARVES),y) += dwarves
```

Comment:
Why don't you add a `CONFIG_TOOLS_BUILD_dwarves` here?
I think this line is very similar to all the previous lines you modified in this file.

Reply:
I also sometimes wondered why I didn't, fixing.